src/Tizen.Sensor/Tizen.Sensor/Plugins/Accelerometer.cs

```diff
@@ -117,6 +117,22 @@ private static int GetCount()
             return count;
         }
 
+        private void GetInitialCount()
```

Comment:
I think "Count" is not a proper word for this function name. is it right?

Reply:
Yes. I will replace it with Data.
"GetInitialData()", will this be fine ?